source/includes/bestpractices.md

```diff
@@ -0,0 +1,20 @@
+# Best Practices
```

Comment:
@Dineshagrawal85 I don't think we need this section?

It seems like best practices for implementing API (on our side), not end users.